src/Service/SvgService.php

```diff
@@ -38,7 +38,7 @@ public function getSvg($name, $width, $height, $viewboxX, $viewboxY, $cssClasses
     {
         $key = sha1(json_encode(func_get_args()));
         if (null === $out = $this->cache->get($key)) {
-            $fileName = sprintf('%s/%s/%s.svg', $this->baseDir, $directory, $name);
+            $fileName = sprintf('%s/%s.svg', $this->baseDir . ($directory ? '/' . $directory : ''), $name);
```

Comment:
Ik zou hier niet `sprintf` en ` . ` mengen.  Je hebt hier een use-case waar alleen `sprintf` niet goed werkt, dan zou ik alles hier met ` . ` doen

i.e. `$fileName = $this->baseDir . ($directory ? '/' . $directory : '') . $name . '.svg';`

Reply:
:pray: ga ik meteen aanpassen! Ik ben sowieso niet echt een voorstander van overmatig gebruik van `sprintf()` :smile: